src/main/java/com/jaspersoft/jasperserver/jaxrs/client/apiadapters/authority/users/SingleUserRequestAdapter.java

```diff
@@ -87,5 +91,123 @@ public void run() {
                 new DefaultErrorHandler());
     }
 
+    @Deprecated
```

Comment:
Please use @deprecated in javadoc (with suggestion) like here: http://www.atetric.com/atetric/javadoc/net.sf.jasperreports/jasperreports/5.6.0/src-html/net/sf/jasperreports/engine/JRExporterParameter.html